jsons/1.12-rel.json

```diff
@@ -35,9 +35,28 @@
       "serverreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "net.minecraftforge:log4j-server-fix:2.8.1",
       "url" : "http://files.minecraftforge.net/maven/",
-      "checksums" : [ "2d9530d0a25daffaffda7c35037b046b627bb171" ],
+      "checksums" : [ "TODO" ],
+      "clientreq":false,
+      "serverreq":true
+    },
+    {
+      "name": "org.jline:jline:3.3.1",
+      "url" : "http://files.minecraftforge.net/maven/",
+      "checksums" : [ "d8a30137fe4ee2246b71b3915baac767d348c5bb" ],
+      "serverreq":true,
+      "clientreq":false
+    },
+    {
+      "name": "net.minecrell:terminalconsoleappender:1.0.0-SNAPSHOT",
```

Comment:
I will release a build of this on Maven central when reviewing is done (in case any chances are necessary). It also needs to be uploaded to Forge maven then.